### ISSUE TITLE
fxLayoutGap & wrap

### ISSUE THREAD

**Issue by MuhammadMatar** [angular/flex-layout#820]
# Bug Report
when using a wrap, on elements in container with layoutGap (as in https://github.com/angular/flex-layout/wiki/fxLayoutGap-API), the last element width will be full width of the container instead of the calculated fxFlex value 
#### What is the expected behavior?
the last element width should be the calculated width 

#### What is the current behavior?
the last element width is even more than 100% of the container width

#### What are the steps to reproduce?
Providing a StackBlitz (or similar) is the *best* way to get the team to see your issue.
(in the link below the last example code)
https://github.com/angular/flex-layout/wiki/fxLayoutGap-API

*  [StackBlitz starter](https://goo.gl/y38GUZ) using latest `npm` nightly release: 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
I'm using 
Agnular - 6.0.3
Angular Material - 6.4.3
Typescript - 2.7.2

#### Is there anything else we should know?
this problem happens when having a 'calculated' fxFlex value (fxFlex="calc(50% - 25px)"), 
when the value is just a parentage (fxFlex="50") the wrap works fine.
In my use I have to add a gap and thus should calculate the fxFlex to be the percentage I give minus the gap width, but the last element (had 3 elements) width is corrupted

**Comment by MuhammadMatar** [comment]
OK I know what was the problem, just needed to add values to shrink and grow to 0, and that will make the size of the last element as it should be (the calculated value)

**Comment by CaerusKaru** [comment]
@MuhammadMatar does that mean this issue can be closed?

**Comment by MuhammadMatar** [comment]
@CaerusKaru 
yes, but think its better to update the example on the site, just to make it more clear 
thank you

**Comment by MuhammadMatar** [comment]
OK :+1:
